articles/azure-app-configuration/howto-import-export-data.md

```diff
@@ -449,23 +449,24 @@ From the Azure portal, follow these steps:
     |--------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------|
-    | Prefix             | Optional. This prefix will be trimmed from the imported keys. A key prefix is the beginning part of a key. Prefixes can be used to manage groups of keys in a configuration store. Prefix will be ignored for feature flags.           | *TestApp:*                 |
-    | From label         | Select an existing label to restrict your export to key-values with a specific label. If you don't select a label, only key-values with the "No label" label will be exported. See note below.                                         | *prod*                    |
-    | At a specific time | Optional. Fill out to import key-values from a specific point in time. This is the point in time of the resource you selected in the previous step. Format: "YYYY-MM-DDThh:mm:ssZ".                                                    | *07/28/2022 12:00:00 AM*  |
-    | Content type       | Optional. Check the box **Override default key-value content types** and select **Key Vault Reference** or **JSON** under **Content type** to state that the imported content consists of a Key Vault reference or a JSON file. | *JSON (application/json)* |
-
-    > [!IMPORTANT]
-    > If the keys you want to export have labels, you must use the command `--label` and enter the corresponding labels. If you don't select a label, only keys without labels will be exported. Use a comma sign (`,`) to select several labels or use `*` to include all labels, including the null label (no label).
+    | Prefix             | Optional. This prefix will be trimmed from each exported key-value's "key" property. A key prefix is the beginning part of a key. Prefixes can be used to manage groups of key-values in a configuration store. Prefix will be ignored for feature flags.           | *TestApp:*                 |
+    | Export as reference | Optional. Check to export key-values to App Service as App Configuration references. [Learn more](../app-service/app-service-configuration-references.md) |
+    | At a specific time | Optional. Fill out to export key-values from a specific point in time. This is the point in time of the resource you selected in the previous step and the version of key-values that existed in the resource at the specified time will be exported. Format: "YYYY-MM-DDThh:mm:ssZ". This field defaults to the current point in time of the resource when left empty.                                                  | *07/28/2022 12:00:00 AM*  |
```

Comment:
The use of the terms "resource" and "version" can be confusing. The "resource" should be replaced with "key-values". We should avoid using "version".

Reply:
Got it. 
Would something like this be better?
`This is the point in time of the key-values in the selected resource. Format: "YYYY-MM-DDThh:mm:ssZ". This field defaults to the current point in time of the key-values when left empty.`

Reply:
Looks good!